projects/fourier/lib/fourier/commands/bundle.rb

```diff
@@ -13,10 +13,41 @@ class Bundle < Base
         required: false,
         aliases: :p,
       )
+      option(
+        :xcode_version,
+        desc: %(
+          The version of Xcode to use to build tuist.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
+          or a file path pointing to the Xcode app bundle to use. (e.g. `--xcode_version="/path/to/Xcode.app"`)
+          Defaults to the currently selected Xcode on the system.
+        ),
+        type: :string,
+        required: false
+      )
+      option(
+        :xcode_version_libraries,
+        desc: %(
+          The version of Xcode to use to build tuist's library dependencies.
+          Can be either a version number (e.g. `--xcode_version="13.2.1"`)
+          or a file path pointing to the Xcode app bundle to use. (e.g. `--xcode_version="/path/to/Xcode.app"`)
+          Defaults to the currently selected Xcode on the system.
+        ),
+        type: :string,
+        required: false
+      )
       def tuist
         output_directory = options[:output]
         output_directory ||= File.expand_path("build", Constants::ROOT_DIRECTORY)
-        Services::Bundle::Tuist.call(output_directory: output_directory)
+
+        xcode_path = Utilities::Xcode.path_to_xcode(options[:xcode_version])
+        xcode_path ||= Utilities::Xcode.current_xcode_version
+        xcode_path_libraries = Utilities::Xcode.path_to_xcode(options[:xcode_version_libraries])
```

Comment:
Maybe we can have a single method returning a tuple instead of duplicating this code multiple times?